Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -993,6 +993,14 @@ script:
     - description: AppSec configuration version.
       name: create_from_version
       required: true
+    - description: Specifies whether the application rules should be migrated to the latest version.
+      name: rule_update
+      required: true
+      auto: PREDEFINED
+      defaultValue: 'True'
```

Comment:
Please set default as false so it won't affect other users

Reply:
The hard coded parameter in the previous version is True, so we will keep it as true for the new version:



    def clone_appsec_config_version(self,
                                    config_id: str,
                                    create_from_version: str) -> Dict:
        """
        Create a new version of security configuration from a previous version
        Args:
            config_id
            versionId

        Returns:
            <Response [204]>
        """
        body = {
            "createFromVersion": int(create_from_version),
            "ruleUpdate": True
        }